src/main/java/org/isf/login/rest/LoginController.java

```diff
@@ -46,10 +46,12 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
-import io.swagger.annotations.Api;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
-@RestController
-@Api(value = "/auth", produces = MediaType.APPLICATION_JSON_VALUE)
+@RestController(value = "/auth")
+@Tag(name = "Auth")
```

Comment:
Move `login` endpoint to appear under the same `Login` tag

```suggestion
@Tag(name = "Login")
```